### PR TITLE
Make connector ops CODEOWNERS of oauth code

### PR DESCRIPTION
**Pull request by sherifnada**
## What
This PR updates code to match reality.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -13,6 +13,11 @@
 /airbyte-integrations/bases/source-acceptance-tests/ @airbytehq/connector-extensibility
 /airbyte-integrations/connector-templates/ @airbytehq/connector-extensibility
 
+# Oauth
+/airbyte-oauth/ @airbytehq/connector-operations
+/airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java @airbytehq/connector-operations
+airbyte-server/src/test/java/io/airbyte/server/handlers/OAuthHandlerTest.java @airbytehq/connector-operations
```
Comment (pedroslopez) [inline .github/CODEOWNERS:19]: nit, I think this is equivalent

```suggestion
/airbyte-server/src/test/java/io/airbyte/server/handlers/OAuthHandlerTest.java @airbytehq/connector-operations
```

**Review by pedroslopez — APPROVED** [review]
Makes sense to me!

**Review by evantahler — APPROVED** [review]
Yes, I think ideally Connector Operations owns oAuth.  I don't think we are fully able to be self-sufficient yet, but this is the goal.

**Comment by sherifnada** [comment]
@evantahler Happy to help on bootstrapping the team's knowledge of oauth & next steps. Will write something up.

**Pull request merged**